code/game/objects/items/robot/robot_parts.dm

```diff
@@ -187,7 +187,7 @@
 				user << "<span class='warning'>The mmi indicates that their mind is currently inactive; it might change!</span>"
 				return
 
-			if(BM.stat == DEAD)
+			if(BM.stat == DEAD || (M.brain && M.brain.damaged_brain))

```

Comment:
The second part of the check should probably be enough but I'm keeping the brain mob stat == DEAD check just to be safe.